x/stakeibc/keeper/msg_server_create_trade_route.go

```diff
@@ -79,17 +79,24 @@ func (ms msgServer) CreateTradeRoute(goCtx context.Context, msg *types.MsgCreate
 	}
 
 	// Register the new ICA accounts
+	tradeRouteId := types.GetTradeRouteId(msg.RewardDenomOnReward, msg.RewardDenomOnHost)
```

Comment:
Second input should be HostDenomOnHost not RewardDenomOnHost

Reply:
good catch!!

Reply:
done